packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -179,12 +179,23 @@ function getOffset(node) {
   };
 }
 
-function getMousePosition(event) {
-  if (event.changedTouches && event.changedTouches[0]) {
-    return {
-      x: event.changedTouches[0].pageX,
-      y: event.changedTouches[0].pageY,
-    };
+function getMousePosition(event, touchId) {
+  if (event.changedTouches) {
+    let touchIndex = 0;
+    for (let i = 0; i < event.changedTouches.length; i += 1) {
+      const touch = event.changedTouches.item(i);
```

Comment:
Needs feature detection since it's not supported in IE 11: https://developer.mozilla.org/en-US/docs/Web/API/TouchList/item#Browser_compatibility